tailwind.config.js

```diff
@@ -5,4 +5,11 @@ const bloomTheme = cloneDeep(require("@sf-digital-services/ui-components/tailwin
 // For example:
 // bloomTheme.theme.colors.white = "#f0f0e9"
 
+// tailwind will automatically purge unused styles when `NODE_ENV` is set to `production`
+bloomTheme.purge = [
+  './src/**/*.tsx',
+  __dirname + '/app/assets/javascripts/**/*.html.slim',
```

Comment:
I don't think we need this line, since anything under /app/assets/javascripts is angular.